src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -1393,8 +1393,7 @@
 							</Border.RenderTransform>
 							<FontIcon x:Name="MultiSelectCheck"
 									  FontFamily="{ThemeResource SymbolThemeFontFamily}"
-									  win:Glyph="&#xE73E;"
-									  not_win:Glyph="&#xE081;"
```

Comment:
I think E73E should be the correct glyph.

Reply:
Yes, you are right. But I don't understand why the default CheckBox style has a different Glyph In contrast to ListViewItem.
Does this make sense?

Reply:
Done

Reply:
Both E73E and E001 appears to be exactly the same glyph in Uno.Fonts.Fluent package. So both should work. But E73E would keep parity with Windows.`

Reply:
Done!
By the way it looks better than was.
Before that, they were larger.
![image](https://github.com/unoplatform/uno/assets/69314237/658319a2-866b-4223-9948-9f9dd61550a1)